packages/charts/react-charting/src/utilities/utilities.ts

```diff
@@ -408,10 +429,14 @@ export function createNumericYAxis(
   // maxOfYVal coming from only area chart and Grouped vertical bar chart(Calculation done at base file)
   const tempVal = maxOfYVal || yMinMaxValues.endValue;
   const finalYmax = tempVal > yMaxValue ? tempVal : yMaxValue!;
-  const finalYmin = yMinMaxValues.startValue < yMinValue ? 0 : yMinValue!;
+  const finalYmin = supportNegativeData
+    ? Math.min(yMinMaxValues.startValue, yMinValue || 0)
+    : yMinMaxValues.startValue < yMinValue
+    ? 0
+    : yMinValue!;
   const domainValues = prepareDatapoints(finalYmax, finalYmin, yAxisTickCount, isIntegralDataset);
   const yAxisScale = d3ScaleLinear()
-    .domain([finalYmin, domainValues[domainValues.length - 1]])
+    .domain([domainValues[0], domainValues[domainValues.length - 1]])
```

Comment:
>domainValues [](http://example.com/codeflow?start=13&length=12)

use finalYmin in case supportNegativeData is false

Reply:
ok